project_config.yml

```
@@ -0,0 +1,4 @@
+catalog_name: mlops_students
+schema_name: mahajan134
+
+target: Outcome
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)